app/src/main/AndroidManifest.xml

```diff
@@ -26,8 +25,8 @@
         </service>
         
         <meta-data
-        android:name="com.google.firebase.messaging.default_notification_channel_id"
-        android:value="@string/default_notification_channel_id" />
+        android:name="com.google.firebase.messaging.default_notification_channel_id" />
+        <!--android:value="@string/default_notification_channel_id"-->
```

Comment:
There was a problem implementing notifications so I commented it out for now

Reply:
Avoid committing commented out code to version control.  This is a no-no, as version control offers better alternatives.
- You're in your own branch, so you can do what you want to do without impacting master.  This gives you a license to experiment.  Instead of commenting out code, delete it.
- You can always go back in history to see what used to be there.